src/ccalls.jl

```diff
@@ -150,34 +151,37 @@ function call_gap_func(func::GapObj, args...; kwargs...)
 end
 
 struct GlobalsType
-    funcs::IdDict{Symbol,Cuint}
 end
 
 Base.show(io::IO,::GlobalsType) = Base.show(io,"table of global GAP objects")
 
 """
     Globals
 
-TODO: describe how this allows accesing any GAP variable or function as `GAP.Globals.VARIABLE_NAME`.
+TODO: describe how this allows accesing any GAP variable or function as
+`GAP.Globals.VARIABLE_NAME`.
 """
-Globals = GlobalsType(IdDict{Symbol,Cuint}())
+Globals = GlobalsType()
```

Comment:
I should mention that this PR also gets rid of our "gvar cache" -- I did some benchmarks and discovered that it actually hurts performance